<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Focos - Don Paco's</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 255);</string>
  </property>
  <widget class="QLabel" name="label_focos_titulo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
font: bold 14pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>ðŸ’¡ Nivel de Luminosidad</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="puerto_com">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
font: bold 20pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
   </property>
   <property name="text">
    <string>Puerto COM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_focos">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffcc00;
color: black;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Encender LEDs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regresar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2a5298;
color: white;
font: 10pt &quot;Arial&quot;;
border-radius: 10px;
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>ðŸ”™ Regresar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_capturar_luz">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffcc00;
color: black;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Capturar Luminosidad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guardar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>490</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2a5298;
color: white;
font: 10pt &quot;Arial&quot;;
border-radius: 10px;
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>GUARDAR DATOS </string>
   </property>
  </widget>
  <widget class="QListWidget" name="listLuminosidad">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>521</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
font: bold 24pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="com">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
font: bold 20pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_conectar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffcc00;
color: black;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>CONECTAR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_estado">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_capturar_luz_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffcc00;
color: black;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>apagar sensor </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
